internal/impl/openai/chat_processor.go

```diff
@@ -58,6 +66,43 @@ To learn more about chat completion, see the https://platform.openai.com/docs/gu
 			service.NewInterpolatedStringField(ocpFieldSystemPrompt).
 				Description("The system prompt to submit along with the user prompt.").
 				Optional(),
+			service.NewIntField(ocpFieldMaxTokens).
+				Optional().
+				Description("The maximum number of tokens that can be generated in the chat completion."),
+			service.NewFloatField(ocpFieldTemp).
+				Optional().
+				Description(`What sampling temperature to use, between 0 and 2. Higher values like 0.8 will make the output more random, while lower values like 0.2 will make it more focused and deterministic.
+
+We generally recommend altering this or top_p but not both.`).
+				LintRule(`root = if this > 2 || this < 0 { [ "field must be between 0 and 2" ] }`),
+			service.NewInterpolatedStringField(ocpFieldUser).
+				Optional().
+				Description("A unique identifier representing your end-user, which can help OpenAI to monitor and detect abuse."),
+			service.NewFloatField(ocpFieldTopP).
+				Optional().
+				Advanced().
+				Description(`An alternative to sampling with temperature, called nucleus sampling, where the model considers the results of the tokens with top_p probability mass. So 0.1 means only the tokens comprising the top 10% probability mass are considered.
```

Comment:
Guess this is also between 0 and 2?

Reply:
Actually between 0-1 

https://community.openai.com/t/what-is-the-top-p-default-value/615875/5

I will add a lint rule - good idea 😄